src/main/java/net/snowflake/ingest/utils/ErrorCode.java

```diff
@@ -36,7 +36,9 @@ public enum ErrorCode {
   CHANNEL_STATUS_INVALID("0028"),
   UNSUPPORTED_DATA_TYPE("0029"),
   INVALID_VALUE_ROW("0030"),
-  MAX_ROW_SIZE_EXCEEDED("0031");
+  MAX_ROW_SIZE_EXCEEDED("0031"),
+  MAKE_URI_FAILURE("0032"),
+  OAUTH_REFRESH_TOKEN_ERROR("0033");
```

Comment:
Let's back this one out for now, that can come in a later PR.

Reply:
Both should be in a later PR since they're not used everywhere else